src/config_creature.h

```diff
@@ -47,7 +47,6 @@ enum CreatureModelFlags {
     CMF_IsLordOTLand     = 0x0008, // simply, Knight
     CMF_IsSpectator      = 0x0010, // simply, Floating spirit
     CMF_IsEvil           = 0x0020, // All evil creatures
-    CMF_NeverChickens    = 0x0040, // Cannot be affected by Chicken (for Avatar)
```

Comment:
shouldn't this one replaced with a dummied out property instead?
disease is ok to be removed cause is at end of list but for that one there is a hole now or I'm misunderstanding it 🤔

Reply:
yes, will add some comments there sure